### PR TITLE
Introduction: replace Nollning

### PR DESCRIPTION
**Pull request by davidwessman**
Removes Nollning namespace and controller
Adds views and actions to introductions

### CODE REVIEW

File: app/views/introductions/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  <% if @news.present? %>
+    <div class="col-md-9 col-sm-12 news">
+      <div class="headline">
+        <h2><%= t('.news') %></h2>
+      </div>
+
+      <% @news.each do |n| %>
+        <%= render '/news/news', news: n %>
+      <% end %>
```
Comment (navro) [inline app/views/introductions/index.html.erb]: We can change this to `render @news` now (we are in the same namespace) :smile:

File: app/views/nollning/nollnings/matrix.html.erb [hunk 2]
```diff
@@ -1,22 +0,0 @@
-<% description(t('.description')) %>
```
Comment (navro) [inline app/views/nollning/nollnings/matrix.html.erb]: Are you working on a new version of this?

**Comment by navro** [comment]
+1

File: app/views/nollning/nollnings/matrix.html.erb (hunk 2 above)
Comment (davidwessman) [inline app/views/nollning/nollnings/matrix.html.erb]: Yep, I got an almost ready version 😄

**Pull request merged**